tests/components/bsblan/snapshots/test_diagnostics.ambr

```diff
@@ -1,12 +1,68 @@
 # serializer version: 1
 # name: test_diagnostics
   dict({
+    'coordinator_data': dict({
+      'state': dict({
+        'current_temperature': dict({
+          'data_type': 0,
+          'desc': '',
+          'name': 'Room temp 1 actual value',
+          'unit': '&deg;C',
+          'value': '18.6',
+        }),
+        'hvac_action': dict({
+          'data_type': 1,
+          'desc': 'Raumtemp’begrenzung',
+          'name': 'Status heating circuit 1',
+          'unit': '',
+          'value': '122',
+        }),
+        'hvac_mode': dict({
+          'data_type': 1,
+          'desc': 'Komfort',
+          'name': 'Operating mode',
+          'unit': '',
+          'value': 'heat',
+        }),
+        'hvac_mode2': dict({
+          'data_type': 1,
+          'desc': 'Reduziert',
+          'name': 'Operating mode',
+          'unit': '',
+          'value': '2',
+        }),
+        'room1_thermostat_mode': dict({
+          'data_type': 1,
+          'desc': 'Kein Bedarf',
+          'name': 'Raumthermostat 1',
+          'unit': '',
+          'value': '0',
+        }),
+        'target_temperature': dict({
+          'data_type': 0,
+          'desc': '',
+          'name': 'Room temperature Comfort setpoint',
+          'unit': '&deg;C',
+          'value': '18.5',
+        }),
+      }),
+    }),
     'device': dict({
       'MAC': '00:80:41:19:69:90',
       'name': 'BSB-LAN',
       'uptime': 969402857,
       'version': '1.0.38-20200730234859',
     }),
+    'entry': dict({
+      'data': dict({
+        'host': '127.0.0.1',
+        'passkey': '1234',
+        'password': 'admin1234',
```

Comment:
Should we redact this?

Reply:
I thought homeassistant automatically redacts this info? Should I add an redact function?

Reply:
You have to redact the data yourself, there's a function for that